config/webpack/webpack.common.js

```diff
@@ -185,7 +186,11 @@ const webpackConfig = ({envFile = '.env', platform = 'web'}) => ({
             },
             {
                 test: /\.css$/i,
-                use: ['style-loader', 'css-loader'],
+                use: ['style-loader', 'file-loader', 'css-loader'],
+            },
+            {
+                test: /\.(woff|woff2|eot|ttf|otf)$/i,
```

Comment:
We definitely want webpack to ignore the `.otf` files, and we just don't need `eot` or `ttf` here

```suggestion
                test: /\.(woff|woff2)$/i,
```